README.md

```diff
@@ -1,4 +1,4 @@
-# Microverse Trials
+# Microverse Trials - -Survey Form
 
 ## Albert Antwi
 ## Newton Ombese
```

Comment:
- Nice efforts there in the README, you've identified the collaborators of the project :+1: , but I would recommend you to add a more detailed README which gives more insights of the project even before someone checks it out. 

You can use this [README template](https://github.com/microverseinc/readme-template). It is well descriptive but kindly customize it to your project. :grin: